Makefile

```diff
@@ -0,0 +1,45 @@
+SHORT_NAME ?= workflow-migration
+
+include versioning.mk
+
+# Enable vendor/ directory support.
+export GO15VENDOREXPERIMENT=1
```

Comment:
This can be removed; it isn't needed/used by deis/go-dev as of https://github.com/deis/docker-go-dev/commit/c47f93ca8cf2e1acdbee16ca9d6d782fedb4f464